packages/react-core/src/components/Content/Content.tsx

```diff
@@ -53,6 +53,8 @@ export interface ContentProps extends React.HTMLProps<HTMLElement>, OUIAProps {
   isPlainList?: boolean;
   /** Flag to indicate the link (or all links within the content) has visited styles applied if the browser determines the link has been visited. */
   isVisitedLink?: boolean;
+  /** Flag to indicate the content has editorial styling */
```

Comment:
Could we add onto this description to mention what "editorial styling" means. An example description would also cover that, but I feel like it's worth having in the prop description as well

Reply:
Added a description of what the prop functionally does. I'm open to other names as well, `editorial` is maybe a little vague or broad.

Reply:
When @edonehoo is available we should get her to look over the copy.  We can make any changes as a follow up PR.